tests/TestData/Providers/EntitiesFilter.php

```diff
@@ -0,0 +1,80 @@
+<?php
+declare(strict_types=1);
+
+namespace StubTests\TestData\Providers;
+
+use StubTests\Model\BasePHPElement;
+use StubTests\Model\PHPClass;
+use StubTests\Model\PHPFunction;
+use StubTests\Model\PHPInterface;
+use StubTests\Model\StubProblemType;
+
+class EntitiesFilter
```

Comment:
Maybe move static methods to `ReflectionTestDataProviders`?

Reply:
I wouldn't do that because these methods also filter entities from stubs, not only from reflection.

Reply:
Sorry, didn't get it right at first glance. Moved all methods exception `getFiltered` to appropriate `ReflectionTestDataProvider` classes